processing_options.go

```diff
@@ -470,6 +472,19 @@ func applyBlurOption(po *processingOptions, args []string) error {
 	return nil
 }
 
+func applyDprOption(po *processingOptions, args []string) error {
+	if len(args) > 1 {
+		return fmt.Errorf("Invalid dpr arguments: %v", args)
+	}
+
+	if d, err := strconv.ParseFloat(args[0], 32); err == nil || (d >= 1 || d <= 8) {
```

Comment:
I don't think we need the upper limit here. If a user wants to generate an image with 100500 DPR, well, as we use to say in Russia, he is his own evil Pinocchio. I think this limit is reasonable only for DPR header.

Reply:
hehe, makes sense.

Reply:
Also, i think it's a good idea to support LowDPI, so let's allow dpi < 1 and > 0.